fades/helpers.py

```diff
@@ -208,3 +210,37 @@ def check_pypi_updates(dependencies):
 
     dependencies["pypi"] = dependencies_up_to_date
     return dependencies
+
+
+def _pypi_head_package(pkg_name):
```

Comment:
In this case we're not checking if the version exists, right? I mean, we're validation that dependency "foobar" exists, but what about if user requested foobar==2.3 and 2.3 doesn't exist?

Reply:
Now it's checking with the version.